ofrak_core/ofrak/core/patch_maker/linkable_binary.py

```diff
@@ -19,6 +19,7 @@
 )
 from ofrak.core.patch_maker.linkable_symbol import LinkableSymbol, LinkableSymbolType
 from ofrak_patch_maker.model import BOM, PatchRegionConfig
+from ofrak_patch_maker.patch_maker import PatchMaker
```

Comment:
Introduces a circular import, as `patch_maker.py` imports from `program.py` which imports from `linkable_binary`. 

We are planning a PR which will remove the PatchMaker dependency on `Program` and other OFRAK classes, so this change will be possible in the future.